### DISCUSSION TITLE
I bought pro but I'm still being prompted to watch ads.

### DISCUSSION THREAD

**Discussion by richmanoski-toast** [Nain57/Smart-AutoClicker#581] [Q&A]
![2024-12-28 15_56_53](https://github.com/user-attachments/assets/c1d5966d-bcfd-4402-82b4-3eba9322be1e)

I purchased it via the Play Store and am running MuMu.

**Comment by Nain57** [comment]
Hi, 

This issue can come from an invalid Google Play Services Cache.

You need to do the following steps:
* Execute the three steps described [here](https://support.google.com/googleplay/answer/9037938?hl=en) in order to correctly clean your cache.
* Reboot your phone and ensure you have an internet connection.
* Start the Google PlayStore app to see if it is correctly functional and then check Klick'r.

Let me know if this solves your problem.
